git-send-email.perl

```diff
@@ -299,7 +304,7 @@ sub signal_handler {
                     "dump-aliases" => \$dump_aliases);
 usage() unless $rc;
 die "--dump-aliases incompatible with other options\n"
-    if !$help and $dump_aliases and @ARGV;
+    if (!$help) and $dump_aliases and @ARGV;
```

Comment:
The parens around the entire expression are optional.  PBP likes explicit parens because some people can't remember their operator precedence.   I don't agree with all PBP rules... Most Perl operator precedence is fairly obvious (except perhaps for 'next', 'last', etc. being considered operators).